.gitignore

```diff
@@ -48,3 +48,6 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+
+# BlackDuck script
+/BlackDuck-Scan.sh
```

Comment:
I think this should go into your `.git/info/exclude` instead because this seems to be a script unique to your workspace.

Reply:
removed